packages/localize/test/LocalizeManager.test.js

```diff
@@ -303,23 +303,23 @@ describe('LocalizeManager', () => {
         throw new Error('did not throw');
       });
 
-      it('throws a warning if the locale set by the user is not a full language locale', async () => {
-        const spy = sinon.spy(console, 'warn');
+      it('throws an error if the locale set by the user is not a full language locale', async () => {
+        const spy = sinon.spy(console, 'error');
```

Comment:
throw is not the same as console.error... let's throw and test throwing here with

```
expect(() => manager.locale = 'nl').to.throw('You can not...');
```

no need to "tinker" with console 👍

Reply:
Aaaaaaah im not actually throwing, thats why it didnt work when I was trying the approach you're proposing. Makes sense now... will change ^^